store/types/utils.go

```diff
@@ -2,6 +2,9 @@ package types
 
 import (
 	"bytes"
+	fmt "fmt"
```

Comment:
Unnecessary, please just keep it as "fmt"

Reply:
thanks